src/backend/joanie/core/serializers/base.py

```diff
@@ -30,3 +30,20 @@ def to_representation(self, instance):
             cache.set(cache_key, representation, cache_ttl)
 
         return representation
+
+
+class ErrorResponseSerializer(serializers.Serializer):
+    """
+    Serializer used to format error responses.
+    """
+
+    details = serializers.CharField(required=True)
+
+    class Meta:
+        fields = ["details"]
+
+    def create(self, validated_data):
+        """Abstract method that should be implemented."""
```

Comment:
should or must… as you want, but the point of adding a create function on an ErrorResponseSerializer is just to avoid NotImplementedError when drf-yasg inspects the code.